webui/react/src/hooks/useModal/Checkpoint/useModalCheckpointRegister.tsx

```diff
@@ -107,7 +107,7 @@ const useModalCheckpointRegister = ({ onClose }: Props = {}): ModalHooks => {
             description: (
               <div className={css.toast}>
                 <p>{`"${versionName || `Version ${selectedModelNumVersions + 1}`}"`} registered</p>
-                <Link path={paths.modelVersionDetails(selectedModelName, response.id)}>
+                <Link path={paths.modelVersionDetails(selectedModelName, response.version)}>
```

Comment:
This is another issue which is in the current version, but:
- I think we should put model.id instead of the `selectedModelName` here
- `selectedModelNumVersions` has flawed logic because if you delete one of the earlier versions, it will still be based on available num_versions. The versionName is showed in the '_ registered' notification line above, so I don't think it's so important. The only area with `selectedModelNumVersions` being more visible is the placeholder in the form field for name

Reply:
None of these points is breaking the specific change of the PR so I will approve it.